NOTICE

```diff
@@ -1,5 +1,12 @@
 Apache Sedona (incubating)
-Copyright 2015-2021 The Apache Software Foundation
+Copyright 2020 and onwards The Apache Software Foundation
```

Comment:
2020 ~ 2022 maybe better?

Reply:
I am not sure... I copied the idea from Apache Spark: https://github.com/apache/spark/blob/master/NOTICE#L2